store/entity-store.html

```diff
@@ -173,8 +173,10 @@
 							return this._store.get(cacheKey).get(lowerCaseEntityId);
 						}.bind(this))
 						.catch(function(err) {
-							this.setError(entityId, resolved, err);
-							return this._store.get(cacheKey).get(lowerCaseEntityId);
+							return this
+								.setError(entityId, resolved, err).then(function() {
```

Comment:
`setError` had the same problem as `update`. It was always a `Promise`, but always ran synchronously, and was used incorrectly. This was failing _handles entity error using promise_